src/client/components/MealsList.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+
+function MealsList() {
+  const [meals, setMeals] = useState([]);
+
+  useEffect(() => {
+    async function fetchMeals() {
+      const response = await fetch("/api/meals");
+      const data = await response.json();
+      setMeals(data);
+    }
+    fetchMeals();
```

Comment:
Good use of async function inside the hook 👍

Reply:
Thank you so much 🫶🏼